lib/wcc/contentful/store/base.rb

```diff
@@ -114,7 +139,7 @@ def apply(filter, context = nil)
 
       protected
 
-      ## naiive implementation
+      # naiive implementation
```

Comment:
might want to add spell checker to our list of linters :trollface:

Reply:
naïve, naive, naiive, I'm indifferent.  I'll change it.